atlp-rwanda/rca-barefoot-team4-bn
# Ch intergrating githubactions 184638221

What does this PR do?
=> This PR is created with updates on the main.yml file which runs github actions on build and test steps of the project.
Description of Task to be completed
=> Github actions were failing on build and this task was to check if actions can pass once build is successful. now build is succeeding however tests are still not working as they are running indefinitely. Once tests are successful the actions will automatically run and succeed.
How should this be manually tested?
=> going to the actions tab of github repository there should be workflows run. some are successful other are not however the actions run automatically so testing this can be done on push and pull-requests
Any background context you want to provide?
=> The actions are necessary to check if the project is successfully running all steps required to work.
What are the relevant pivot tracker story?
=> there is no pivotal task related to this PR however since the actions are needed to display actions badge in the readme file it can be related to the pivot task #184638221
Screenshots (if appropriate) no screenshots as the actions run on github after PR is created.
